bin/cyhy-import

```diff
@@ -79,6 +81,27 @@ def has_intersections(db, nets, filename, owner):
         print_intersections(intersections)
         return True
 
+def has_restricted_ips(nets):
+    geo_loc_db = GeoLocDB()
+    restricted_dict = defaultdict(lambda: netaddr.IPSet())
```

Comment:
The other use of `netaddr.IPSet()` at https://github.com/cisagov/cyhy-core/blob/127fdccc1c21181f2fb60a145a7bd75ff1199ad4/bin/cyhy-import#L114 needs to be done as well. This is needed because of the change to the import.

```suggestion
    restricted_dict = defaultdict(lambda: IPSet())
```